specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1022,7 +1020,7 @@ components:
         toelichtingBewaarder:
           type: "string"
           description: "Toelichtende tekst bij een onroerende zaak van de bewaarder. De bewaarder is iemand die bij het Kadaster werkt. Hij schrijft stukken in in de openbare registers en de basisregistratie Kadaster conform de Kadasterwet."
-    AmbtelijkCorrectiegegevenID:
+    PortefeuilleID:
```

Comment:
in imKad en ook volgens camleCase "PortefeuilleId" (dus de D van ID als kleine letter)